protocol-designer/src/load-file/migration/8_0_0.ts

```diff
@@ -67,6 +67,20 @@ export const migrateFile = (
     },
   ]
 
+  const migrateCommands = (
+    v7Commands: ProtocolFileV7<{}>['commands']
+  ): ProtocolFile['commands'] => {
+    return v7Commands.filter(
+      v7Command =>
+        !(
+          v7Command.commandType === 'loadLabware' &&
+          v7Command.params.labwareId === 'fixedTrash'
+        )
```

Comment:
what you have totally works, it feels a bit cleaner imo to filter it this way as well! Either way though.

```suggestion        
          v7Command.commandType === 'loadLabware' &&
          v7Command.params.labwareId !== 'fixedTrash'       
```

Reply:
won't this filter out all non-loadLabware commands?

Reply:
ooh good call, i forgot this is looking through all commands instead of just load commands. Ya you can keep it as is!

Reply:
no prob!